src/App.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { Container } from 'react-bootstrap';
 
-import MainPage from './pages/MainPage';
+import MainRoute from './pages/MainRoutePage/MainRoute';
```

Comment:
in this branch u also have  MainPage folder. Remove it if u don't use

Reply:
inside `pages ` folder u have a real page, but not a wrapper with routes and header. U can move this logic in `App.js` file